Test/Tasty/ExpectedFailure.hs

```diff
@@ -145,6 +151,11 @@ expectFail' reason = wrapTest (fmap change)
                   | otherwise      = t ++ "\n" ++ s
     comment = maybe "" (mappend ": ") reason
 
+    goldenAccepted r
+        | "Accepted the new version" `isPrefixOf` resultDescription r = True
+        | "Golden file did not exist; created" == resultDescription r = True
+        | otherwise = False
```

Comment:
> Uff, comparing strings, really? 

I wish there is a more strutured way to distinguish "golden" successes from regular ones, but I couldn't find one!